src/EnrollChoicesController.js

```diff
@@ -87,6 +87,10 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
       },
 
       initialize: function (options) {
+        if (options.appState.get('webauthnAbortController')) {
```

Comment:
I think this should be handled in a similar way to how we cancel polling when we switch from Okta Verify to another factor. There is a `factorSwitched` event that is fired in [FactorDropdownOptions#L31](https://github.com/okta/okta-signin-widget/blob/master/src/views/mfa-verify/dropdown/FactorsDropDownOptions.js#L31) that we listen to in [Factor#315](https://github.com/okta/okta-signin-widget/blob/master/src/models/Factor.js#L315), so we can stop polling. Sounds like we can do something similar?

Reply:
nice! I reused this and moved away from appstate as well.

Reply:
curious why you cannot extend `src/models/Factor.js` as Mau mentioned but have to listen to a new event when click back?

Reply:
@haishengwu-okta  I am reusing the event in verify flow. But during the enroll when back is clicked I don't see any event being triggered.

Reply:
I meant `backToFactors`. looks like enroll and challenge flow need different event?

Reply:
I think this is right, since those 2 events are different:
- `factorSwitched` is the event we have when users select a new factor in the beacon dropdown. This is only in verify flows
- `backToFactors` is the new event created in this PR, this is only in enroll flows (there is no beacon dropdown in enroll flows)

Reply:
yeah in verify we switchFactors and enroll we can go back to factorlist.